### ISSUE TITLE
[feature]: Enable users to select and switch between HTTP/1.1, HTTP/2, and HTTP/3 for API requests.

### ISSUE THREAD

**Issue by Gowtham2003** [hoppscotch/hoppscotch#4385]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

This feature would allow users to specify which HTTP version they want to use when sending API requests. By providing the option to switch between HTTP/1.1, HTTP/2, and HTTP/3, This would help to test applications on various http versions.

### Why should this be worked on?

Many developers need to test their APIs across different HTTP versions to ensure compatibility. 
Currently, without the ability to switch HTTP versions, users may struggle to assess how their APIs perform under various conditions. 
This feature would enhance Hoppscotch's functionality, making it very useful for testing applciations.

**Comment by iRittikSharma** [comment]
@Gowtham2003  I would like to work on this issue could you please assign it to me

**Comment by Gtyrande** [comment]
Yes, this function is very important when user want to call API restricted to HTTP 2.0
It will show "Could not send request"

In postman, there is a setting tab to switch HTTP version, hope Hoppscotch can support version switch too
